Maple2.Server.Game/Manager/Items/InventoryManager.cs

```diff
@@ -333,6 +339,41 @@ public class InventoryManager {
         return true;
     }
 
+    public bool MailItem(Item item) {
```

Comment:
I would move this to `ItemManager` since it's not really inventory related anymore. Also you don't need the lock since it doesn't mutate inventory.